blocks/color-palette/style.scss

```diff
@@ -114,8 +114,8 @@ $color-palette-circle-spacing: 14px;
 	width: 200%;
 	height: 200%;
 	position: absolute;
-		top: -50%;
-		left: -50%;
+	top: -50%;
```

Comment:
cc @mtias , he likes this convention 😉 I'll leave it to him to defend.

Reply:
Yeah, I could see the logic but it's not used elsewhere in the project & it will throw an error in `stylelint-config-wordpress` once g7g is ready to merge into core.

As an aside, I'll set up a ticket to add `stylelint-config-wordpress` as a dependency to the project if it doesn't exist already.

Reply:
It doesn't have it currently, I attempted to do this in #1151, at the time the stylelint SCSS rules included in `stylelint-config-wordpress` needed to be updated, this remains the case today, an audit of the current rules included in the SCSS configuration of `stylelint-config-wordpress` is required.